optional_plugins/golang/setup.py

```diff
@@ -23,12 +25,22 @@
 else:
     packages = find_namespace_packages(include=["avocado_golang"])
 
-VERSION = open("VERSION", "r", encoding="utf-8").read().strip()
+BASE_PATH = os.path.dirname(__file__)
+with open(os.path.join(BASE_PATH, "VERSION"), "r", encoding="utf-8") as version_file:
+    VERSION = version_file.read().strip()
+
+
+def get_long_description():
+    with open(os.path.join(BASE_PATH, "README.rst"), "rt", encoding="utf-8") as readme:
+        readme_contents = readme.read()
+    return readme_contents
+
 
 setup(
     name="avocado-framework-plugin-golang",
-    description="Avocado Plugin for Execution of Golang tests",
     version=VERSION,
+    description="Avocado Plugin for Execution of Golang tests",
+    long_description=get_long_description(),
```

Comment:
If we are adding `long_description` IMO, based on [documentation](https://packaging.python.org/en/latest/guides/making-a-pypi-friendly-readme/), we should also add `long_description_content_type` attributes. This is related to all commits from this PR.